dashboard/README.md

```diff
@@ -1,198 +1,161 @@
-    │   │   ├── index.js
-    │   │   ├── loadingReducer.js
-    │   │   ├── navbarReducer.js
-    │   │   ├── overviewReducer.js
-    │   │   ├── tableOfContentReducer.js
-    │   │   ├── toastReducer.js
-    │   │   └── userProfileReducer.js
-    │   ├── store
-    │   │   └── store.js
-    │   ├── utils
-    │   │   ├── axiosInstance.js
-    │   │   ├── dateFunctions.js
-    │   │   ├── filterDataset.js
-    │   │   ├── helper.js
-    │   │   └── routeConstants.js
-    │   ├── App.css
-    │   ├── App.js
-    │   ├── App.test.js
-    │   ├── index.css
-    │   ├── index.js
-    │   ├── logo.svg
-    │   ├── reportWebVitals.js
-    │   └── setupTests.js
-    ├── config-overrides.js
-    ├── jsconfig.json
-    ├── jsdoc.json
-    ├── package.json
-    ├── package-lock.json
-    └── README.md
+ .
+ ├── public									# HTML entry
+ ├── server 								# endpoint configuration
+ └── src
+     ├── actions							# redux actions
+     ├── assets
+     │   ├── constants						# constants used in code
+     │   ├── images							# images
+     │   └── logo							# Pbench logos
+     ├── modules
+     │   ├── components						# functional components
+     │   └── containers						# parent components
+     ├── reducers							# redux reducer
+     ├── store								# redux store
+     └── utils								# utility functions
 ```
 
+  
+
 ## Assets
 
+  
+
 Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
 
+  
+
 ## Prerequisites
 
+  
+
 ## Install Node JS
-Refer to https://nodejs.org/en/ to install nodejs
+
+Refer [https://nodejs.org/en/] (Node.js) official website
+
+  
 
 ## Install create-react-app
+
 Install create-react-app npm package globally. This will help to easily run the project and also build the source files easily. Use the following command to install create-react-app
 
+  
+
 ```bash
+
 $ npm install -g create-react-app
+
 ```
 
-## Cloning and Running the Application in local
+  
+
+## Cloning and Running the Application in local  
+
+- Clone the [Pbench code](https://github.com/distributed-system-analysis/pbench) into local
+- Install all the npm packages
 
-Clone the project into local
-Install all the npm packages. Go into the project folder and type the following command to install all npm packages
+Go into the project folder and type the following command to install all npm packages 
 
 ```bash
+
 $ npm install
+
 ```
-In order to run the application Type the following command
+  
+
+`server/index.js` contains references to datastores and metadata required to visualize data in the dashboard.
+
+  
+
+Create `.env` file in the root directory and declare the environment variable
+
+  
 
 ```bash
+
+$ PBENCH_SERVER=http://test_domain.com
+
+```
+
+where `PBENCH_SERVER` should point to a real server.
+
+In order to start the express server and run the application use the following command 
```

Comment:
Yeah, maybe that's an inappropriate technicality, although it's a phrase I often use and which I suspect will be meaningful and helpful to most Javascript developers.

We're talking about the Javascript endpoints "passthrough" reflector, (`dashboard/server/index.js`) which is built on the Javascript "express" package (think of it as a Javascript Flask) and "axios" (a Javascript "requests") to redirect the `GET /api/v1/endpoints` from `localhost:3000` to a designated remote Pbench Server when running the dashboard in local developer mode.

Reply:
> We're talking about

Yeah, I know...but, I'm not sure that it will be meaningful to future readers (or future _me_...).

I meant my comment as a rhetorical question, intended to prompt the author to expand the referenced text.  I should have been clearer...my bad.